# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/types.h-x86_64.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Mismatch with kernel headers ...

  - Rene Rebe <rene@exactcode.de>

--- dietlibc/include/asm/types.h	2005-08-24 16:57:45.000000000 +0000
+++ dietlibc/include/asm/types.h	2005-08-24 21:53:49.000000000 +0000
@@ -25,7 +25,7 @@
 typedef int64_t __s64;
 #endif
 
-#if defined(__alpha__)
+#if defined(__alpha__) || defined(__x86_64__)
 typedef unsigned long __kernel_size_t;
 #else
 typedef unsigned int __kernel_size_t;
